Content.Shared/Interfaces/GameObjects/Components/Interaction/IThrowCollide.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Map;
+
+namespace Content.Shared.Interfaces.GameObjects.Components
+{
+    public interface IThrowCollide
```

Comment:
What is this interface?

It seems to be both being hit by things and htiting things?

Seems a bit messy (lots of "were we the thing hit or were we thrown" checks (`if(entity.Thrown==Owner)`))

Reply:
hmm, should I make this two different methods on the same interface?
One for when being hit, and one for when hitting something else

Reply:
I'd advocate for interfaces like this to be single use.

i.e. `IThrowCollisionHit` and `IThrowCollisionHitBy` (names up for discussion)

Reply:
Two methods is good, yes

Reply:
Sounds good! Done.